events/birthday.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: training
+title: Foreman's 7th Birthday Events
+---
+
+<div class='col-md-6 pull-right'>
+<img src="/static/images/birthday.png" title="IHappy Birthday" />
+<p style="font-size:8px;text-align:right">By <a href="//commons.wikimedia.org/wiki/User:Hendrike" title="User:Hendrike">Hendrike</a> - <span class="int-own-work" lang="en"><a href='https://commons.wikimedia.org/w/index.php?curid=3259208'>Own work</a></span>, <a title="Creative Commons Attribution-Share Alike 3.0" href="http://creativecommons.org/licenses/by-sa/3.0/">CC BY-SA 3.0</a></p>
+</div>
+
+## Foreman's 7th Birthday
+
+**It's our brithday!** On July 13th, we'll be 7 years old. In combination with
+the 1.12 release, including the long-awaited Puppet 4 support, we think this
+worth celebrating! 
+
+## Time for a party!
+
+We're planning a series of regional events though out the second half of July
+and early August. Below you'll find details of what's currently planned - do
+check back for updates. All the events will of course be available on the
+[events page](/events) and the [ICS calendar](/events/all.ics).
+
+If you want to organise a Foreman event yourself, please do! We'll be happy to
+support you, just drop a line to the [community
+guy](mailto:community@emeraldreverie.org).
+
+### Foreman @ Techmeetup Edinburgh (13th July 2016)
+
+The birthday party kicks off in Edinburgh, where our very own community guy,
+[Greg Sutcliffe](https://twitter.com/gwmngilfen) will be giving a Foreman talk
+to the [Techmeetup](http://techmeetup.co.uk) community. This is likely to be an
+overview talk, covering the basics of Foreman, what's new in 1.12, and it's
+awesome community.
+
+This talk should be recorded and livestreamed (links will be added here once we
+know them) so do tune in and get involved.
+
+### Foreman @ NETWAYS Nürnberg (21st July 2016)
+
+One evening of Foreman not enough? How about a proper half-day event with
+hands-on time and multiple speakers? Now, that's a party!
+
+The awesome folks at NETWAYS are hosting a halfday Foreman event, kicking off
+with 2+ hours of hands-on time with Foreman - perfect for your first steps with
+Foreman, to get help from some of the experienced people present, or just to do
+some in-person hacking.
+
+We'll then break for coffee before having some talks from users and
+contributors, and then head out for some food.
+
+Further links and schedule will be added once it's all confirmed.
+
```

Comment:
@dgoetz We can add a copy of the schedule here and a link to NETWAYS etc, once it's all finalised - but this should be enough for you to link to in your blog post right?